index.html

```diff
@@ -40,6 +40,27 @@ <h1>
             <button type="submit">Rechercher</button>
           </form>
         </section>
+        <section id="Filter">
+          <p>
+            Filtes
+          </p>
+          <div>
+            <input type="checkbox" name="economy" id="economy">
```

Comment:
Why input ??? It's juste icone with text

Reply:
Je voulais mettre une sélection multiple sur mes filtres , cela regroupe les filtres en fonction de la demande de utilisateur  exemple : si la personne souhaite apporter son animal et aussi que se soit économique, elle pourra sélectionner ses deux catégories mais pour éviter de compliquer j'ai remis une liste avec les texte à l'intérieur